src/fontra/views/editor/editor.js

```diff
@@ -1780,6 +1847,174 @@ export class EditorController {
     });
   }
 
+  canAddAnchor() {
+    return this.sceneModel.getSelectedPositionedGlyph()?.glyph.canEdit;
+  }
+
+  async doAddAnchor() {
+    const point = this.sceneController.selectedGlyphPoint(this.contextMenuPosition);
+    const { anchor: tempAnchor } = await this.doAddEditAnchorDialog(undefined, point);
+    if (!tempAnchor) {
+      return;
+    }
+
+    const newAnchor = {
+      name: tempAnchor.name ? tempAnchor.name : "anchorName",
+      // x will be specified individually for each layer
+      y: tempAnchor.y ? tempAnchor.y : Math.round(point.y),
+    };
+    const relativeScaleX =
```

Comment:
This will cause problems when xAdvance is zero (not unusual for combining accents)

Reply:
Fixed it.

Reply:
I don't like the fix: you still divide by zero where it makes no sense.